File: hello_world.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

// Exibir "Hello, World!"
int main() {
    printf("Hello, World!\n");

    // Gerar um número aleatório
    srand(time(NULL));
    int randomNum = rand();
    printf("Random number: %d\n", randomNum);

    // Mostrar a data e hora atual
    time_t now = time(NULL);
    printf("Current date and time: %s", ctime(&now));

    return 0;
}

File: hello_world.pas
program BasicTasks;
uses sysutils;

begin
  // Exibir "Hello, World!"
  writeln('Hello, World!');

  // Gerar um número aleatório
  randomize;
  writeln('Random number: ', random(100));

  // Mostrar a data e hora atual
  writeln('Current date and time: ', DateTimeToStr(Now));
end.
